src/corner-dialog/src/CornerDialog.js

```diff
@@ -147,11 +148,7 @@ export default class CornerDialog extends PureComponent {
     onCancel: close => close(),
     onConfirm: close => close(),
     onCloseComplete: () => {},
-    position: {
-      position: 'fixed',
-      bottom: 50,
-      right: 50
-    }
+    position: AbsolutePosition.BOTTOM_LEFT
```

Comment:
I think we expect consumers to pass the string or constant (also a string) for `BOTTOM_LEFT`. We would then lookup the top/left/right/bottom props based on that string value, else default to `'BOTTOM_RIGHT'`.

Also the default prop should still be `BOTTOM_RIGHT`.